packages/@atjson/renderer-plain-text/src/index.ts

```diff
@@ -13,4 +24,48 @@ export default class PlainTextRenderer extends Renderer {
     let text = yield;
     return text.join("");
   }
+
+  *LineBreak() {
+    return "\n";
+  }
+
+  *Heading() {
+    let item = yield;
+    return item.join("") + "\n\n";
+  }
+
+  *Blockquote() {
+    let item = yield;
+    return item.join("") + "\n\n";
+  }
+
+  *Pullquote() {
+    let item = yield;
+    return item.join("") + "\n\n";
+  }
+
+  *ListItem() {
+    let item = yield;
+    return item.join("");
+  }
+
+  *List(annotation: List) {
+    let items: string[] = yield;
+    if (annotation.attributes.type === "bulleted") {
+      items = items.map((item) => {
+        return `- ${item}`;
+      });
+    } else if (annotation.attributes.type === "numbered") {
+      let startsAt = annotation.attributes.startsAt ?? 1;
+      items = items.map((item, index) => {
+        return `${startsAt + index}. ${item}`;
+      });
+    }
+    return items.join("\n") + "\n";
```

Comment:
You probably want to render two new lines after a list:
```
Paragraph before a list is separated by two lines

- List item
- List item

Next paragraph is separated by two lines
```

Besides being more consistent, I see another reason to do this: right now a nice feature of how you've implemented the plaintext renderer is if you pasted its output into a markdown editor, basic things like paragraphs and lists carry over. To keep this behavior you'd want the extra line.